src/diffusers/pipelines/free_noise_utils.py

```diff
@@ -70,6 +168,9 @@ def _enable_free_noise_in_block(self, block: Union[CrossAttnDownBlockMotion, Dow
                     motion_module.transformer_blocks[i].load_state_dict(
                         basic_transfomer_block.state_dict(), strict=True
                     )
+                    motion_module.transformer_blocks[i].set_chunk_feed_forward(
```

Comment:
Do we always need chunked feed forward set when enabling free noise? Might be overkill no?

Reply:
This is only there to carry forward the chunk FF behaviour if and only if it was already enabled in the BasicTransformerBlock. Basically, if it was not enable in BTB, `motion_module.transformer_blocks[i]._chunk_size` would be None leading to default behaviour of no chunking. If it was enabled in BTB, it would by default carry forward to FreeNoiseTransformerBlock